app/_src/kubernetes-ingress-controller/reference/ingress-to-gateway-migration.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Migrating Ingress to Gateway
+type: reference
+purpose: |
+  Which ingress annotations are migrated to Gateway API features?
+---
+
+As the name states, the {{site.kic_product_name}} was born as an ingress controller.
+Starting from the early stage of the Gateway API development, the {{site.kic_product_name}}
+began supporting the Gateway API as well, as an alternative to the ingress resources.
+Since the Gateway API graduation, happened in late 2023, the {{site.kic_product_name}}
+promoted the Gateway API as the preferred way of configuring the {{site.base_gateway}}
+in Kubernetes.
+
+## The IngressToGateway tool
+
+Kong contributed to the kubernetes-sigs project [`ingress2gateway`](https://github.com/kubernetes-sigs/ingress2gateway)
+by creating a Kong provider able to convert ingress resources into Gateway ones.
+The `Ingress2Gateway` tool provides a unique `print` command that gets ingress resources
+and displays the Gateway API equivalent. The output of such an operation can be either
+directly applied to the cluster, or into a file, to have a new set of yaml files
+containing the converted configuration.
+
+## Supported Ingress features and annotations
+
+The following is a list of all the ingress features, along with the Gateway API
+equivalent support.
+
+Kong ingresses provides an extensive set of [annotations](../references/annotations.md).
+Below is the list of the features supported by the Ingress2Gateway's Kong provider.
+Such a list will be improved over time.
+
+| Annotation name | Conversion |
+|-----------------|-------------------------|
+| REQUIRED [`kubernetes.io/ingress.class`](#kubernetesioingressclass) | `gateway.spec.gatewayClassName` |
+| [`konghq.com/methods`](#konghqcommethods) | `httpRoute.spec.rules[*].matches[*].method` |
+| [`konghq.com/headers.*`](#konghqcomheaders) | `httpRoute.spec.rules[*].matches[*].headers` |
+| [`konghq.com/plugins`](#konghqcomplugins) | `httpRoute.spec.rules[*].matches[*].headers` |
+
+## Features
+
+### kubernetes.io/ingress.class
```

Comment:
```suggestion
### `kubernetes.io/ingress.class`
```

Don't we want code syntax highlighting since this is a known const? (and similarly do the rest like so?)

Reply:
I didn't do that for consistency, as we have a [page](https://docs.konghq.com/kubernetes-ingress-controller/latest/references/annotations/) with tons of such annotations unformatted. If we want to do code highlight, I think we should do it consistently (and probably in a dedicated PR).